llvm/test/tools/sycl-post-link/device-globals/test_global_variable_many_modules_no_dev_img_scope.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: sycl-post-link --device-globals --split=source -S %s -o %t.files.table
-; RUN: FileCheck %s -input-file=%t.files_0.ll --check-prefix CHECK-MOD0
-; RUN: FileCheck %s -input-file=%t.files_1.ll --check-prefix CHECK-MOD1
-; RUN: FileCheck %s -input-file=%t.files_2.ll --check-prefix CHECK-MOD2
+; RUN: opt -passes=compile-time-properties %t.files_0.ll -S -o %t.files_0.ll && FileCheck %s -input-file=%t.files_0.ll --check-prefix CHECK-MOD0
+; RUN: opt -passes=compile-time-properties %t.files_1.ll -S -o %t.files_1.ll && FileCheck %s -input-file=%t.files_1.ll --check-prefix CHECK-MOD1
+; RUN: opt -passes=compile-time-properties %t.files_2.ll -S -o %t.files_2.ll && FileCheck %s -input-file=%t.files_2.ll --check-prefix CHECK-MOD2
```

Comment:
Since compile time properties handling is not a part of `sycl-post-link` duties anymore, I don't think that it should be necessary to check for such handling here.

I would suggest to either run compile time properties pass on the file and replace it with the processed version or to only launch the pass once and pass the output to `sycl-post-link`

Perhaps we may even completely omit those checks, because I would expect that stuff to be tested in compile time properties pass unit-tests

Reply:
Ran ahead of time as the `sycl-post-link` tool splits the file and checks each individually. Running the pass first in the pipeline removes the `CHECK` lines.